test/e2e/components/vl-multi-select.js

```diff
@@ -1,6 +1,136 @@
-const { VlElement } = require('vl-ui-core').Test;
+const { VlSelect } = require('vl-ui-select').Test;
+const { Pill } = require('./pill');
+const { By } = require('selenium-webdriver');
+
+class VlMultiSelect extends VlSelect {
```

Comment:
ik zou een class of object maken met methodes/properties 
- getText
- getValue
- isSelected

En dan op de multiselect:
- getSelectedItems
- getUnselectedItems
- getAllItems
- select(Item)
- unselect(Item)

Dat lijkt mij ongeveer de API te moeten zijn.